### PR TITLE
Add admin user to docker group for Kubernetes

### PR DESCRIPTION
**Pull request by JiangtianLi**
Fix https://github.com/Azure/acs-engine/issues/124 where docker ps needs sudo to work in Kubernetes. Add admin user to docker group for Kubernetes.

### CODE REVIEW

File: parts/kubernetesagentcustomdata.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ runcmd:
 - /usr/bin/curl -sSL --retry 12 --retry-delay 10 https://storage.googleapis.com/kubernetes-release/release/{{{kubectlVersion}}}/bin/linux/amd64/kubectl > /usr/local/bin/kubectl
 - chmod +x /usr/local/bin/kubectl
 - mkdir -p /etc/kubernetes/manifests
+- sudo usermod -aG docker {{{username}}}
```
Comment (colemickens) [inline parts/kubernetesagentcustomdata.yml]: no need for `sudo` here

**Comment by JiangtianLi** [comment]
@colemickens Thanks, removed sudo

@anhowe Thanks, added changed templates and test files

**Comment by colemickens** [comment]
Needs a rebase, but can be merged whenever.

**Comment by JiangtianLi** [comment]
@colemickens Did a rebase and push again.

**Comment by colemickens** [comment]
Merge away :)

**Comment by colemickens** [comment]
fwiw, my guess is that it was the newline, since the breaking PR removed the one that this one re-inserts: https://github.com/Azure/acs-engine/pull/294/files#diff-762e5e591524857e49023ea5248e1df0L1362

anyway, looks green!

**Comment by JiangtianLi** [comment]
I see. Good catch. It is likely I resolved conflict in Atom and Atom did the automatic newline removal.

**Pull request merged**